### ISSUE TITLE
Discord Profile Pic as Skin!

### ISSUE THREAD

**Issue by iPro36** [Stasadance/Germs.io#9]
Let the players, use their **discord profile pic** as their **skin** in the game.

Something like this:
![capture](https://user-images.githubusercontent.com/38023207/38277973-40be848e-3768-11e8-80e0-49e12112c573.jpg)

*Note: Since people might use **NSFW** photos as their skins, you might want to consider having a moderation team that will **approve/disapprove** the pic.

**Comment by Stasadance** [comment]
Moderation team won't cut it in the long term. Has to be another way to do it or someway to reduce the amount of work a mod team would have to do

**Comment by iPro36** [comment]
How about adding a **button** on the main menu that only moderators can see and that can lead to another **window** where all they would have to is click yes or no for the pic, but that would be some work on your part.

**Comment by Stasadance** [comment]
Thats the plan but for 1000 players submitting skins all day long its gonna add up quick. I don't think mods will want to do this for very long before it becomes tedious. Getting and maintaining 10 people on the discord server to do it won't be hard but I'd have to have the system on discord to make it easier on myself, and I dont even know where to start with that, maybe a skin submit bot or something..

**Comment by iPro36** [comment]
Yeah, discord bot sounds like a good idea and also to limit the work the mods need to do you can limit the number of skins that can be submitted by each user...
